src/mail/view/MailViewerHeader.ts

```diff
@@ -485,37 +487,24 @@ export class MailViewerHeader implements Component<MailViewerHeaderAttrs> {
 						? // If we have exactly one attachment, just show the attachment
 						  this.renderAttachmentContainer(viewModel, attachments)
 						: // Otherwise, we show the number of attachments and its total size along with a show all button
-						  [
-								m(
-									".flex.center-vertically.click.flex-grow.ml-between-s.mt-xs",
-									{
-										role: "button",
-										"aria-pressed": String(this.detailsExpanded),
-										tabindex: TabIndex.Default,
-										onclick: () => (this.filesExpanded = !this.filesExpanded),
-										onkeydown: (e: KeyboardEvent) => {
-											if (isKeyPressed(e.key, Keys.SPACE, Keys.RETURN)) {
-												this.filesExpanded = !this.filesExpanded
-												e.preventDefault()
-											}
-										},
-									},
-									[
-										m(
-											"",
-											lang.get("attachmentAmount_label", { "{amount}": attachmentCount + "" }) +
-												` (${formatStorageSize(totalAttachmentSize)})`,
-										),
-										m(Icon, {
-											icon: BootIcons.Expand,
-											style: {
-												fill: theme.content_fg,
-												transform: this.filesExpanded ? "rotate(180deg)" : "",
-											},
-										}),
-									],
-								),
-						  ],
+						  m(ExpanderButton, {
```

Comment:
this now has a different size of the icon so it doesn't look like the other expander above it